src/lib.rs

```diff
@@ -4388,27 +4397,35 @@ static FNLIST_320: CK_FUNCTION_LIST_3_2 = CK_FUNCTION_LIST_3_2 {
     C_UnwrapKeyAuthenticated: Some(fn_unwrap_key_authenticated),
 };
 
+/// PKCS#11 reserved name for the standard interfaces
 static INTERFACE_NAME_STD_NUL: &str = "PKCS 11\0";
 
+/// Holds pointers to v2.40 interfaces
 static INTERFACE_240: CK_INTERFACE = CK_INTERFACE {
     pInterfaceName: INTERFACE_NAME_STD_NUL.as_ptr() as *mut u8,
     pFunctionList: &FNLIST_240 as *const _ as *const ::std::os::raw::c_void,
     flags: 0,
 };
 
+/// Holds pointers to v3.0 interfaces
 static INTERFACE_300: CK_INTERFACE = CK_INTERFACE {
     pInterfaceName: INTERFACE_NAME_STD_NUL.as_ptr() as *mut u8,
     pFunctionList: &FNLIST_300 as *const _ as *const ::std::os::raw::c_void,
     flags: 0,
 };
 
+/// Holds pointers to v3.2 interfaces
```

Comment:
nit: I think each of these is a pointer to a single interface, not multiple interfaces:

```suggestion
/// Holds pointers to v2.40 interface
static INTERFACE_240: CK_INTERFACE = CK_INTERFACE {
    pInterfaceName: INTERFACE_NAME_STD_NUL.as_ptr() as *mut u8,
    pFunctionList: &FNLIST_240 as *const _ as *const ::std::os::raw::c_void,
    flags: 0,
};

/// Holds pointers to v3.0 interface
static INTERFACE_300: CK_INTERFACE = CK_INTERFACE {
    pInterfaceName: INTERFACE_NAME_STD_NUL.as_ptr() as *mut u8,
    pFunctionList: &FNLIST_300 as *const _ as *const ::std::os::raw::c_void,
    flags: 0,
};

/// Holds pointers to v3.2 interface
```

Reply:
ack